.wvr-ud/static-assets/styles.css

```diff
@@ -148,4 +148,2292 @@ main.loading #preview {
 .feather-plus-circle,
 .open .feather-minus-circle {
   display: inline-block;
-}
\ No newline at end of file
+}
+html {
+  height: 100%;
+}
+body {
+  font-size: .875rem;
+  height: 100%;
+  overflow: hidden;
+}
+
+.bi {
+  width: 16px;
+  height: 16px;
+  vertical-align: text-bottom;
+}
+
+/*
+* Sidebar
+*/
+
+.sidebar {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 100; /* Behind the navbar */
+  padding: 0;
+  box-shadow: inset -1px 0 0 rgba(0, 0, 0, .1);
+}
+
+.sidebar-sticky {
+  position: -webkit-sticky;
+  position: sticky;
+  top: 48px; /* Height of navbar */
+  height: calc(100vh - 48px);
+  padding-top: .5rem;
+  overflow-x: hidden;
+  overflow-y: auto; /* Scrollable contents if viewport is shorter than content. */
+}
+
+.sidebar .nav-link {
+  font-weight: 500;
+  color: #333;
+}
+
+.sidebar .nav-link .bi {
+  margin-right: 4px;
+  color: #999;
+}
+
+.sidebar .nav-link.active {
+  color: #007bff;
+}
+
+.sidebar .nav-link:hover .bi,
+.sidebar .nav-link.active .bi {
+  color: inherit;
+}
+
+.sidebar-heading {
+  font-size: .75rem;
+  text-transform: uppercase;
+}
+
+/*
+* Navbar
+*/
+
+.navbar-brand {
+  padding-top: .75rem;
+  padding-bottom: .75rem;
+  font-size: 1rem;
+  background-color: rgba(0, 0, 0, .25);
+  box-shadow: inset -1px 0 0 rgba(0, 0, 0, .25);
+}
+
+.navbar .form-control {
+  padding: .75rem 1rem;
+  border-width: 0;
+  border-radius: 0;
+}
+
+.form-control-dark {
+  color: #fff;
+  background-color: rgba(255, 255, 255, .1);
+  border-color: rgba(255, 255, 255, .1);
+}
+
+.form-control-dark:focus {
+  border-color: transparent;
+  box-shadow: 0 0 0 3px rgba(255, 255, 255, .25);
+}
+
+/*
+* Utilities
+*/
+
+.border-top { border-top: 1px solid #e5e5e5; }
+.border-bottom { border-bottom: 1px solid #e5e5e5; }
+
+main {
+  min-height: 100%;
+}
+
+#editor,
+#scss-editor,
+#source-editor {
+  height:400px;
+  width: 100%;
+}
+
+main #preview {
+  filter: blur(0rem);
+  transition: .25s filter ease-in-out;
+}
+
+main.loading #preview {
+  filter: blur(.25rem);
+  transition: .25s filter ease-in-out;
+}
+
+.copy-btn {
+  position: fixed;
+  bottom: 25px;
+  right: 26px;
+}
+
+#active-component-name {
+  position: fixed;
+  right: 25px;
+  font-size: 1.5em;
+}
+
+.bi-folder-plus,
+.closed .bi-folder-minus {
+  display: none;
+}
+
+.bi-folder-minus,
+.closed .bi-folder-plus {
+  display: inline-block;
+}
+
+.feather-minus-circle,
+.open .feather-plus-circle  {
+  display: none;
+}
+
+.feather-plus-circle,
+.open .feather-minus-circle {
+  display: inline-block;
+}
+
+/*** OVERRIDES ***/
+
+html {
+  height: 100%;
+}
+body {
+  font-size: .875rem;
+  height: 100%;
+  overflow: hidden;
+}
+
+.bi {
```

Comment:
This class is also duplicated with same values.

Reply:
See above